### DISCUSSION TITLE
i3 bar with gaps

### DISCUSSION THREAD

**Discussion by sabrehagen** [i3/i3#5631] [Q&A] [answered]
Is there a way to add gaps around i3bar, just like windows have outer gaps? I would like gaps around i3bar to be equal to the current outer gaps of the windows.

This is the result I want, found in polybar docs: https://raw.githubusercontent.com/adi1090x/files/master/polybar-themes/previews/docky/md.gif

**Comment by okraits** [comment, marked as answer]
No, it's not possible.

**Reply by sabrehagen** [reply]
@Airblader can confirm?

**Reply by okraits** [reply]
Do you see any setting for configuring gaps for i3bar under https://i3wm.org/docs/userguide.html#_configuring_i3bar? :wink:

**Reply by sabrehagen** [reply]
No, that's why I asked here. I have seen other solutions where people create multiple bars and set one to transparent to simulate a gap.
